### DISCUSSION TITLE
Nvim and clang

### DISCUSSION THREAD

**Discussion by MattBystrin** [clangd/clangd#899] [Q&A] [answered]
Hello !
I'm using clang with neovim 0.5.1 . I'm woking with gcc compiler and if I want jump to `size_t` definition clangd move me to /urs/lib/llvm-11/lib ... instead of gccs' folders. How can I fix this ?

**Comment by HighCommander4** [comment, marked as answer]
Assuming you have a compilation database, and its commands start with the path to your gcc, passing `--query-driver=<path>` with a matching path to your gcc as a command line argument to clangd should fix this.

See [this question in the docs](https://clangd.llvm.org/troubleshooting#cant-find-standard-library-headers-map-stdioh-etc) for more details.
